packages/components/switch/package.json

```diff
@@ -17,6 +17,7 @@
     "directory": "packages/components/switch"
   },
   "dependencies": {
+    "@radix-ui/react-id": "^1.0.0",
```

Comment:
There's an agreement of using fixed version for external dependencies

Reply:
https://github.com/adevinta/spark/wiki/Agreements#dependencies-versionning-on-packagejson